### ISSUE TITLE
No sources in jar after adding dependency

### ISSUE THREAD

**Issue by s3r6** [ihsanbal/LoggingInterceptor#52]
Are they extracted to 3.0.0 artifact?

**Comment by zeatual** [comment]
Are you added maven repositories?
````
allprojects {
	repositories {
		...
		maven { url 'https://jitpack.io' }
	}
}
````

**Comment by s3r6** [comment]
Yes of cource, the dep will not be downloaded instead, but it is.
Here is the jar:
![default](https://user-images.githubusercontent.com/4716175/47835065-88e2fc00-ddbb-11e8-91af-ffae687b744f.png)
and it has no sources, clicking Download has no effect:
![default](https://user-images.githubusercontent.com/4716175/47835083-a2844380-ddbb-11e8-855d-023e32fa7acd.png)
I attached repository sources manually as a quickfix, but they should be provided with artifact for easier usage and javadoc popups
